src/WebServer.ino

```diff
@@ -218,6 +218,7 @@ void WebServerInit()
   web_server.on(F("/i2cscanner"),    handle_i2cscanner);
   #endif
   web_server.on(F("/json"),          handle_json);     // Also part of WEBSERVER_NEW_UI
+  web_server.on(F("/csv"),           handle_csvval);
```

Comment:
This is part of another PR, which is not (yet) merged.

Reply:
That's correct, i've branched from that PR for my convenience. 
I hope that PR will be merged too, but in meantime i can rebase it on current mega if you really want.